packages/next-swc/crates/next-core/src/next_manifests/client_reference_manifest.rs

```diff
@@ -171,15 +173,28 @@ impl ClientReferenceManifest {
                         ManifestNodeEntry {
                             name: "*".to_string(),
                             id: (&*ssr_module_id).into(),
-                            chunks: ssr_chunks_paths.clone(),
+                            chunks: if runtime == NextRuntime::Edge {
+                                vec![]
```

Comment:
Add a comment that this is already added to the middleware chunks so already evaluated

Reply:
done